docs/Installing.asciidoc

```diff
@@ -120,16 +138,16 @@ zypper in openQA-worker
 dnf install openqa-worker
 --------------------------------------------------------------------------------
 
-To allow workers to access your instance, you need to log into
-openQA as operator and create a pair of API key and secret. Once you
-are logged in, follow the link 'manage API keys' in the top right
-corner. Click the 'create' button to generate +key+ and +secret+. There is
-also a script available for creating an admin user and an API key+secret
-pair non-interactively, +/usr/share/openqa/script/create_admin+, which can
-be useful for scripted deployments of openQA. Copy and paste the key and
-secret into +/etc/openqa/client.conf+ on the machine(s) where the worker
-is installed. Make sure to put in a section reflecting your webserver URL.
-In the simplest case, your +client.conf+ may look like this:
+To allow workers to access your instance, you need to log into openQA as
+operator and create a pair of API key and secret. Once you are logged in, in the
+top right corner, is the user menu, follow the link 'manage API keys'.  Click
```

Comment:
Or is it a comma? Can't zoom on my smartphone ;-)
If it's a comma then my opinion is: You have too many commas :-)